src/lib.rs

```diff
@@ -72,6 +72,9 @@ mod sys;
 #[path="windows.rs"]
 mod sys;
 
+#[cfg(not(windows))]
+pub use sys::SharedChildExt;
```

Comment:
Or put the current `unix.rs`/`windows.rs` under `sys/` as a module with conditional inclusion in `sys/mod.rs` and then have `unix.rs` at the top expose this extra trait.